resources/views/users/panel.blade.php

```diff
@@ -69,7 +69,7 @@
             </dd>
           </dl>
           @if($user->profiles()->exists())
-          <a class="btn btn-secondary btn-sm" href="{{ route('profiles.show', [$user->pea]) }}">Profile</a>
```

Comment:
The change in this file also shouldn't be a part of this PR, since it's already in the other PR.